crates/frontend/src/pages/ContextOverride/context_override.rs

```diff
@@ -163,7 +163,7 @@ fn modal_component(handle_submit: Rc<dyn Fn()>) -> impl IntoView {
                         current_tenant,
                         overrides(),
                         context_conditions(),
-                        dimensions().unwrap().expect("resource not loaded"),
+                        dimensions().unwrap(),
```

Comment:
Lets keep expect instead of unwrap? Can we not do unwrap here anyway?

Reply:
Why not trigger an error toast instead of doing an unwrap in such places.

Reply:
I refactored the code little bit which made `dimensions` type from Option<Result<...>> to Option<...>, so had to remove an extra `expect`, anyway I will replace unwrap with expect.

@mahatoankitkumar , will add the toasts everywhere in one shot in a separate PR.

Reply:
i have added a function in utils called `unwrap_option_or_default_with_error`, may be you can you it.

Reply:
@mahatoankitkumar , let me plan this out, anyway I have to add the toasts everywhere, will do it, can we leave this currently, not really needed in this PR.